Get-OfficeMRU.ps1

```diff
@@ -0,0 +1,71 @@
+<#
+.SYNOPSIS
+Get-OfficeMRU.ps1 acquires Microsoft Office MRU from registry
+and reformats on the target as tsv output.
+
+.NOTES
+Next line needed by Kansa.ps1 for proper handling of this script's data
+OUTPUT tsv
```

Comment:
This output directive is deprecated. Kansa.ps1 now takes a command line argument that specifies how output will be formatted, iff your script returns PowerShell objects. All collector scripts must return objects in order for this to work.

Your collector doesn't return PS objects. Take a look at just about any other collector script to see an example.

Try/Catch blocks are only needed to handle terminating errors, that is to say, errors that would cause the script to terminate prematurely. Try/Catch won't do anything for non-terminating errors. Unfortunately, I don't know of a good way to determine if something will cause a terminating error or a non-terminating error, you have to make synthetic tests to figure it out, but as it is below, I suspect your Try/Catch blocks are not going to do what I think you think they are going to do. Kansa.ps1 and many of the collector scripts are riddled with examples of how to do both Try/Catch and non-terminating error handling.